### PR TITLE
Replace Planet Vala link with Vala 2022 wesbite link

### LINKED ISSUE #363: Add link to new vala.dev website
I think that it should be the destination that the "About Vala" link takes you to.

What do you all think?

### PR DESCRIPTION
**Pull request by colinkiama**
Fixes #363

### CODE REVIEW

**Review by lewisgoddard — APPROVED** [review]

**Comment by lewisgoddard** [comment]
Should we change the planet vala link that's on every page instead? That hasn't been updated since 2019.

**Comment by colinkiama** [comment]
Yes, that would make more sense. However, the globe icon is a bit vague isn't it? Couldn't that be changed to o?

**Comment by lewisgoddard** [comment]
What to?

**Comment by colinkiama** [comment]
Something more like this network icon 

image

It's more obvious that it's something related to the web.

**Comment by colinkiama** [comment]
Never mind about the icon, that could be changed in the future.

**Comment by colinkiama** [comment]
This pull request now just replaces the Planet Vala link with the new Vala website.

File: data/index.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ function get_title ($page) {
     </form>
     <a class="title" href="/index.htm"><img alt="Valadoc" src="/images/logo.svg"/></a>
     <ul>
-      <li><a href="https://planet.vala-project.org/" target="_blank" title="Planet Vala"><i class="fa fa-globe"></i></a>
+      <li><a href="https://vala.dev/" target="_blank" title="Vala Official Website"><i class="fa fa-globe"></i></a>
```
Comment (benwaffle) [inline data/index.php:82]:
```suggestion
      <li><a href="https://vala.dev/" target="_blank" title="Vala Official Website"><i class="fa fa-house"></i></a>
```

**Review by lewisgoddard — APPROVED** [review]

**Pull request merged**